### ISSUE TITLE
What to do about multiple ISBNs?

### ISSUE THREAD

**Issue by khanson679** [lingbib/lingbib#8]
Books often have multiple ISBNs. Is one good enough?

If so, does it matter which? This could be a problem, as databases often just several numbers with no further details.

**Comment by adamliter** [comment]
Yes, this is a good point. One that's worth thinking a little bit about, and perhaps if this working group forms, they might have insight to offer on the issue.

My intuition is that perhaps having the ISBN for the original hardback and paperback editions would be good (i.e., two ISBNS, which can be comma-separated in the `isbn` field, I think). However, one thing worth investigating is whether reprints have the same page numbers. If not, then we'll potentially want to have multiple entries of the same work for each time it was printed with different page numbers. I'm not sure what else might bear on our answer to this question.

**Comment by khanson679** [comment]
Are you referring to page numbers in `@incollection`, or something else?

**Comment by adamliter** [comment]
No, I'm talking about when the book is reprinted. I wonder whether reprints of a book have different page numbers. If they do, then it's worth having multiple entries in the repository since page numbers will vary based on which version of the book the person is citing. But if page numbers do not change when a book is reprinted, then we can probably just have an entry with the ISBNs of the first hardback and paperback editions.

**Comment by khanson679** [comment]
Yikes, that kind of thing sounds like a nightmare to determine.

--Kenneth

**Comment by adamliter** [comment]
I would assume that page numbers are the same in a reprint, but I don't know that for sure. We could just try to check a couple of books and see. Or maybe we could ask someone who would actually know.
